Sources/Common/Store/SdkConfig.swift

```diff
@@ -108,6 +108,12 @@ public struct NotificationServiceExtensionSdkConfig {
     public var logLevel: CioLogLevel
     /// See `SdkConfig.autoTrackDeviceAttributes`
     public var autoTrackDeviceAttributes: Bool
+    // property is used internally so disable swiftlint rule
+    /**
+     Used internally at Customer.io to override some information in the SDK when the SDK is being used
+     as a wrapper/bridge such as with ReactNative.
+     */
+    public var _sdkWrapperConfig: SdkWrapperConfig? // swiftlint:disable:this identifier_name
```

Comment:
Do we expect our customers to set value of _sdkWrapperConfig as `react native` or `flutter` ? Is this going to be a "mandatory" configuration specially for react native or flutter ? If yes, then we should add it in our docs